src/api/search/src/bin/validation.js

```diff
@@ -82,7 +82,7 @@ const advancedQueryValidationRules = [
  */
 const validateQuery = () => {
   return async (req, res, next) => {
-    const rules = req.baseUrl === '/' ? queryValidationRules : advancedQueryValidationRules;
```

Comment:
Probably want to remove the variable on line 38 as well so you get ESLint fixed as well

Reply:
We need it for the advanced search... Maybe I should just comment it all out. Technically we are going to remove out the basic search

Reply:
But honestly we can probably fix this now anyways. The req.baseURL isn't working like it was when I built and tested it :(

Reply:
Do you think if I make the function take a parameter it will be better? Do we see ourselves expanding to more search types?

Reply:
If you need it, I would just comment it out. Since it's probably going to come back, I would mention that it's gonna be commented out until the fix comes with a link to whatever issue is filed that deals with it

Reply:
I would avoid using code comments to do project management, since they get forgotten and lost over time.  The best move would be to rip out the code, delete what we don't use, and file an issue that has the code you want to replace it with later, linking to some old commit that shows what you did.  If/when we fix this in the future, we can resurrect it from git/issues.